src/app/cli/src/cli_commands.rs

```diff
@@ -541,3 +541,19 @@ where
 
     Ok(result_refs)
 }
+
+// ToDo add validation for DatasetRefPattern and DatasetRef expresion
+fn validate_many_dataset_wildcards<I>(
+    _catalog: &dill::Catalog,
+    dataset_refs: I,
+) -> Result<Vec<DatasetRefPattern>, CLIError>
+where
+    I: IntoIterator<Item = DatasetRefPattern>,
+{
+    let mut result_refs = Vec::new();
+    for dataset_ref in dataset_refs.into_iter() {
+        result_refs.push(dataset_ref);
```

Comment:
`validate_many_dataset_refs` function exists only because we sometimes allow CLI to work in the "multi-tenant" mode. This is a mode we only use for testing, where inside of your local workspace you can created datasets that belong to different "accounts". These accounts are fake, but it gives us ability to test some multi-tenant features in the comfort of local CLI. 

`validate_many_dataset_refs` simply checks that `account/dataset` reference format is used only in multi-tenant workspaces.

In your case with `validate_many_dataset_wildcards` since `DatasetRefPattern` is a superset of `DatasetRef` type and can be `my.datasets.%` or `alice/my.datasets.%` it also can be single- or multi-tenant, thus we should replicate the same check for it.